configuration/ingress/README.md

```diff
@@ -18,12 +18,17 @@ Specify this with the `ingress.ingressClassName` key in the Howso Platform value
 
 See the [documentation here](./domain-customization/README.md) for information about configuring the domains used by the Howso Platform.
 
-## TLS Certificates**:
+## TLS Certificates
 
 These [examples](../../custom-ingress-cert/README.md) show how to configure the Howso Platform to use custom TLS certificates.
 
+## TLS Termination at the application
+
+This [example](../../application-tls-termination/README.md) shows how to configure the Howso Platform to encrypt traffic between the ingress and the application. 
+
```

Comment:
Does there need to be a separate writeup for mutual TLS within the cluster; including generating client certificates; and including system components like NATS?  (This was the thing that struck me reading this writeup, that it covers one specific set of links "inside the box" but there's still a lot of unencrypted HTTP and NATS traffic if you only do those steps.)

Reply:
Yep - that is coming in a seperate PR